test/src/main.rs

```diff
@@ -55,8 +51,21 @@ fn main() {
     let worker_count = value_t!(matches, "concurrent", usize).unwrap_or_else(|err| err.exit());
     let vendor = value_t!(matches, "vendor", PathBuf).unwrap_or_else(|_| current_dir());
     let fail_fast = !matches.is_present("no-fail-fast");
-    let quiet = matches.is_present("quiet");
     let report = !matches.is_present("no-report");
+    let verbose = matches.is_present("verbose");
+
+    let _ = {
+        let filter = if !verbose {
+            env::var("RUST_LOG").unwrap_or("info".to_string())
+        } else {
+            format!(
+                "{},{}",
+                env::var("RUST_LOG").unwrap_or("info".to_string()),
+                "ckb_test=trace"
```

Comment:
- I prefer to use `RUST_LOG` directly.
  It's easy to create confusion by mixing `RUST_LOG` and `--verbose`.
- Don't hardcode the package name, it could be introduced from an environment variable in compile-time.

Reply:
The purpose of `--verbose` is offer an easy way to set `ckb_test` log level as `trace`, and it's also can be treated as a flag of this log level. `RUST_LOG` is a good way to do this, so I make `--verbose` as a default `false` option argument.

And also, `--verbose` only add `ckb_test=trace` into `RUST_LOG`, the only condition to create confusion is when set `ckb_test=xxx` in `RUST_LOG` at same time, but I don't think this is a question since you already want to show `verbose` log.